examples/vitals/pull_vitals.py

```diff
@@ -53,13 +53,80 @@
 x = 0
 output = {}
 while(x < num):
-    parent = str(pw.devices[x].device[0].device.componentParentDin.value)
-    name = str(pw.devices[x].device[0].device.din.value)
-    print("Device %d: %s " % (x, pw.devices[x].device[0].device.din.value))
-    print("   - %s" % pw.devices[x].device[0].device.componentParentDin.value)
-    print("     - %s" % pw.devices[x].device[0].device.din.value)
+    device = pw.devices[x].device.device
+    parent = str(device.componentParentDin.value)
+    vitals = pw.devices[x].vitals
+    alerts = pw.devices[x].alerts
+
+    name = str(device.din.value)
+    print("Device %d: %s " % (x, name))
+
+    # e.STSTSM = "STSTSM",
```

Comment:
I don't have a direct translation for these types. If they are known I would love to add friendly values to represent them to the callers.